### PR TITLE
Post about the tests we run on the toolkit

### PR DESCRIPTION
**Pull request by Br3nda**

### CODE REVIEW

File: _posts/2019-07-30-Testing-Content.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
+---
+title: Testing our Toolkit
+date: 2019-07-31
+authors:
+  - brenda-wallace
+tags:
+  - open source
+  - tech
+excerpt: ""
```
Comment (JacOng17) [inline _posts/2019-07-30-Testing-Content.md]:
```suggestion
excerpt:
```

File: README.md [hunk 2]
```diff
@@ -122,17 +122,17 @@ external_link_title: Github
...
 excerpt: ""
 ---
 ```
 
 ## Accessiblity Tests
 
-We must meet the [New Zealand GovernmentNew Zealand Government - Web Accessibility Standard 1.1](https://www.digital.govt.nz/standards-and-guidance/nz-government-web-standards/web-accessibility-standard-1-1/) on this toolkit.
+We must meet the [New Zealand Government - Web Accessibility Standard 1.1](https://www.digital.govt.nz/standards-and-guidance/nz-government-web-standards/web-accessibility-standard-1-1/) on this toolkit.
 
 We used automated testing to detect some in-accessible content. These tests are gatekeepers. When the content added doesn't comply, the PR will have a failed test. Receiving automated messages that your work didn't pass can be frustrating at first, but compliance is necessary and becomes easier with practice.
```
Comment (JacOng17) [inline README.md]:
```suggestion
We used automated testing to detect some inaccessible content. These tests are gatekeepers. When the content added doesn't comply, the PR will have a failed test. Receiving automated messages that your work didn't pass can be frustrating at first, but compliance is necessary and becomes easier with practice.
```

**Comment by Marita2010** [comment]
Looks good.

**Comment by Br3nda** [comment]
renaming the file

**Pull request merged**